oak-query-spi/pom.xml

```diff
@@ -39,7 +39,6 @@
           <configuration>
             <instructions>
               <Import-Package>
-                <!-- OAK-7182 -->${guava.osgi.import},
```

Comment:
I think the entire Import-Package instruction can be removed. Importing everything that is required is the [default behavior](https://felix.apache.org/documentation/subprojects/apache-felix-maven-bundle-plugin-bnd.html#_default_behavior).

Reply:
Yep. I'm planning to a cleanup if all POMs once we're done.